Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue10865.xaml.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using System.Linq;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.CarouselView)]
+#endif
+    [Preserve(AllMembers = true)]
+    [Issue(IssueTracker.Github, 10865, "[Bug] CarouselView Throws when removing an item", PlatformAffected.Android)]
+	public partial class Issue10865 : TestContentPage
+	{
+		public Issue10865()
+		{
+#if APP
+			InitializeComponent();
+#endif
+		}
+
+		protected override void Init()
+		{
+
+		}
+    }
+
+    [Preserve(AllMembers = true)]
+    public class Issue10865ViewModel
```

Comment:
This test could be automated.

Reply:
In fact, since it's just a test of what the items sources return when they are empty, it could just be a platform test; no UI test necessary.

Reply:
Also, the test instructions do not match the instructions from the original issue.

Reply:
Right, added unit test.

Reply:
Unit test looks good, can we just remove this screen entirely?

Reply:
Sure, I can remove it, but, why not leave the screen next to the test?

Reply:
Because it's one more screen that we have to maintain as things change, and one more class/screen to load every time we load the solution or test.